static/src/javascripts/boot.js

```diff
@@ -63,14 +63,7 @@ const go = () => {
             if (!willShow) storage.local.set('gu.hasSeenPrivacyBanner', true);
         });
 
-        if (
-            config.get('switches.auConsent', false) ||
-            config.get('tests.useAusCmpVariant') === 'variant'
-        ) {
-            cmp.init({ pubData, country: await getLocale() });
-        } else {
-            cmp.init({ pubData, isInUsa: isInUsa() });
-        }
+        cmp.init({ pubData, country: geolocationGetSync() });
```

Comment:
Why did we move back to `geolocationGetSync`? Seems like a merge error in .

Reply:
Videos, for example, will not work outside of defined editions… PR for fix: #23322